server/controllers/user.controller.js

```diff
@@ -115,11 +138,191 @@ const UserController = function () {
     }
   };
 
+  var updateUserByEmail = async function (req, res, next) {
+    try {
+      const user = await User.findOne({
+        where: {
+          email: req.body.email,
+        },
+      });
+
+      if (!user) {
+        res.status(404);
+        throw new Error(`no user was found with email ${user.email}`);
+      }
+
+      delete req.body.email;
+
+      req.params = { id: user.id };
+
+      const response = await updateUserById(req, res, next);
+    } catch (error) {
+      next(error);
+    }
+  };
+
+  var sendEmail = async function (recipientEmail, subject, emailContent) {
+    const apiKey = process.env.BREVO_API_KEY;
```

Comment:
nit: can we be specific as to why API Key this is, `brevoAPIKey`?
```suggestion
    const brevoAPIKey = process.env.BREVO_API_KEY;
```

Reply:
Changed!